app/ide-desktop/lib/content/globals.d.ts

```diff
@@ -0,0 +1,15 @@
+/** @file Globals defined only in this module. */
+
+declare global {
+    // These are top-level constants, and therefore should be `CONSTANT_CASE`.
+    /* eslint-disable @typescript-eslint/naming-convention */
+    /** Whether the  */
+    /** Whether the application may have the local backend running. */
+    const SUPPORTS_LOCAL_BACKEND: boolean
+    /** Whether the application supports deep links. This is only true when using
+     * the installed app on macOS and Windows. */
+    const SUPPORTS_DEEP_LINKS: boolean
```

Comment:
i don't think so - we also need to know whether it's being run as `ide watch` or as `Enso.app`, since it's not possible to handle deep links on `ide watch`

Reply:
Then this should be written in the docs! :)